src/ImageViewer.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.custom.imageviewer.ImageViewer" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Image viewer</name>
+    <description>Change a number value using a slider</description>
+    <icon/>
+    <properties>
+        <property key="source" type="enumeration" defaultValue="imageUrlAttribute">
+            <caption>Data source</caption>
+            <category>Data source</category>
+            <description></description>
+            <enumerationValues>
+                <enumerationValue key="systemImage">System image</enumerationValue>
+                <enumerationValue key="imageUrlAttribute">Dynamic URL</enumerationValue>
+                <enumerationValue key="staticUrl">Static URL</enumerationValue>
+                <enumerationValue key="staticImage">Static Image</enumerationValue>
+            </enumerationValues>
```

Comment:
Naming can be improved, and options reduced... Perhaps: context, URL, static.
Whether to use the URL attribute or the static field can be determined based on which option is provided, with the URL attribute taking precedence.